web/components/remove-swap-liquidity.tsx

```diff
@@ -54,180 +58,294 @@ export const RemoveSwapLiquidity: React.FC = ({ match }) => {
-                <path fillRule="evenodd" d="M5.293 7.293a1 1 0 011.414 0L10 10.586l3.293-3.293a1 1 0 111.414 1.414l-4 4a1 1 0 01-1.414 0l-4-4a1 1 0 010-1.414z" clipRule="evenodd"></path>
-              </svg>
-
-              <div className="mt-1 relative rounded-md shadow-sm">
-                <div className="absolute inset-y-0 left-0 pl-3 flex items-center pointer-events-none">
-                  <span className="text-gray-500 sm:text-sm">
-                  </span>
-                </div>
-                <input type="text" name="tokenYAmount" id="tokenYAmount"
-                  value={tokenYAmount}
-                  disabled={true}
-                  className="focus:ring-indigo-500 focus:border-indigo-500 block w-full pl-7 pr-12 sm:text-sm border-gray-300 rounded-md" placeholder="0.00" />
-                <div className="absolute inset-y-0 right-0 flex items-center">
-                  <label htmlFor="tokenYAmount" className="sr-only">Token Y</label>
-                  <select id="tokenY" name="tokenY"
-                      value={tokenY}
-                      className="focus:ring-indigo-500 focus:border-indigo-500 h-full py-0 pl-2 pr-7 border-transparent bg-transparent text-gray-500 sm:text-sm rounded-md">
-                    <option>{tokenY}</option>
-                  </select>
-                </div>
-              </div>
-              <p className="font-light ml-2">Balance: {balanceSelectedTokenY} {tokenY}</p>
-              <p className="font-light ml-2">1 {tokenY} = ≈{currentPrice} {tokenX}</p>
-
-              <div className="mt-5 ml-5 sm:flex sm:items-start sm:justify-between">
-                <div className="max-w-xl text-sm text-gray-500">
-                  <div className="mt-5 sm:mt-0 sm:flex-shrink-0 sm:flex sm:items-right">
-                    <button type="button" onClick={() => removeLiquidity()} className="inline-flex items-right px-4 py-2 border border-transparent shadow-sm font-medium rounded-md text-white bg-indigo-600 hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500 sm:text-sm">
-                      Supply
+                  <div className="mt-4 lg:flex lg:items-start lg:justify-between">
+                    <h4 className="uppercase font-semibold text-xs text-gray-700 ">Price</h4>
+                    <div className="mt-3 sm:mt-0 space-y-0.5 lg:text-right text-sm text-gray-500">
+                      <p>
+                        1 {tokenX.name} = {tokenXPrice} {tokenY.name}
+                      </p>
+                      <p  >
+                        1 {tokenY.name} = {tokenYPrice} {tokenX.name}
+                      </p>                    
+                    </div>
+                  </div>
+                  
+                  <div className="mt-4 lg:flex lg:items-center lg:flex-1 lg:space-x-2">
+                    <button
```

Comment:
We probably need some feedback/message/disabled state like we did for other buttons (see Swap and Add liquidity pages)